dashboards/proposal.md

```diff
@@ -0,0 +1,74 @@
+
+* **Dashboard Layout** where the notebook author can drag/drop and resize notebook cells in a grid layout to dynamically create the web application within Jupyter.
+* **Dashboard Transformation** where the notebook author can convert the notebook document into a web app that executes the same code as the origina notebook and retains the desired application layout. Packaging of the web app assets includes:
+	* Static web resources necessary for operation of the dashboard frontend
+	* Configuration for directing the frontend to an appropriate kernel execution environment (e.g., [tmpnb](https://github.com/jupyter/tmpnb), [kernel gateway](https://github.com/jupyter-incubator/proposals/pull/3) )
+	* Assets to ease deployment of the dashboard frontend using common cloud technologies (e.g., Dockerfile, Cloud Foundry manifest)
+
+After publishing the additional prototype, improvements will occur within the incubator project in order to mature the features for later integration into Jupyter Notebook or promotion as other mainline components.
+
+## Audience
+
+* Notebook users who wish to switch to a dashboard view for easier interaction with widgets in their notebooks (e.g., during data exploration)
+* Notebook users who wish to layout and present their results in a dashboard view
+* Notebook users who wish to share interactive dashboard views with other notebook users when passing around notebook documents
+* Non-notebook users who want to interact with findings from analyses done in notebooks without running Jupyter Notebook themselves
+* New Notebook users who wish to use it as a light-weight application composition environment
+
+## Other options
+
+* Any number of commercial and open source dashboarding applications exist (e.g., Tableau). Some have the ability to produce dashboards specifically from notebooks (e.g., Mathematica Cloud, Apache Zeppelin, Databricks).
+* The traditional approach of doing analysis in notebooks and then stepping outside to create a traditional web application is always an option.
+* One could use [thebe](https://github.com/oreillymedia/thebe) or an equivalent JS library to manually create a web application that uses Jupyter kernels for code execution. This project automates this otherwise manual effort. 
+
+## Integration with Project Jupyter
+In accordance with the [New Subproject Process](https://github.com/jupyter/governance/blob/master/newsubprojects.md#incubation-of-subprojects), this incubation proposal pertains to an idea that has yet to be vetted with the community. 
+
+A successful incubation process should provide justification for the inclusion of the dashboard layout and conversion features into the baseline [Jupyter Interactive Notebook](https://github.com/jupyter/notebook). In addition, successful incubation should drive the creation of new Jupyter components that enable the robust execution of deployed notebook-dashboards, as well as other next generation Jupyter client applications.
```

Comment:
Just as a side note,  it might not be in https://github.com/jupyter/notebook repo itself, but under the jupyter org. it can perfectly be a dependency which is it's own repo, and just automatically(or optionally) installed with the jupyter main package. It's technically different, same in principle, but I don't want that to bite you if you have any deliverable.